### ISSUE TITLE
pipenv --clear option doesn't exist anymore?

### ISSUE THREAD

**Issue by asmaier** [pypa/pipenv#2630]
##### Issue description
According to the documentation an option `--clear` should exist for the command `pipenv`, see https://docs.pipenv.org/#pipenv . But on Mac OS X 10.13.4 with pipenv installed via homebrew
I do get
```
$ pipenv --version
pipenv, version 2018.7.1
$ pipenv --clear
Error: no such option: --clear
```

##### Expected result
I expected 
`$ pipenv --clear`
to clear the caches as described in the documentation.

**Comment by mlouielu** [comment]
Fixed at `d6dd412f40195ceffa2f521155c0c70d75d29f4e`

**Comment by techalchemy** [comment]
(It was actually introduced at that commit and it hasn't been released yet)
